kanselarij-vlaanderen/frontend-kaleidos
# KAS-1676: Goedkeuren en sluiten

# KAS-1676: Goedkeurren en sluiten van agenda

In deze PR hebben we 1 actie gemaakt voor het goedkeuren en sluiten van een agenda..

## What has been done:
- Er is een nieuwe functionaliteit voorzien om een agenda goed te keuren en af te sluiten.
- De `agenda-approve-service` is voorzien van een nieuwe endpoint om de agenda goed te keuren.

## Screenshots:
![image](https://user-images.githubusercontent.com/11557630/88407332-4274d280-cdd2-11ea-9de9-77dbc534fa75.png)

## To Test:
- check de brach uit voor de `frontend`, `agenda-approve-service`
- update `docker-compose.override` en override de agenda-approve-service.

## medecredits:
- @woutersf
